lib/testsuite/nightwatch-inspector/websocket-server.js

```diff
@@ -0,0 +1,60 @@
+const {WebSocketServer} = require('ws');
+const Utils = require('../../utils');
+const {Logger} = Utils;
+
+module.exports = class Websocket {
+  constructor() {
+    this.finishCallback = null;
```

Comment:
why is the `finishCallback` required?

Reply:
It is not required in my case. I followed the same concept as `process-listener`

Reply:
remove if not required

Reply:
removed